src/main/cpp/VideoState.h

```diff
@@ -227,18 +261,48 @@ class VideoState {
 
 	int step;
 
+	int newSpeed_req;
+	float last_speed;
+	float pts_speed;
+
 #if CONFIG_AVFILTER
 	int vfilter_idx;
+	const char **vfilters_list = NULL;
```

Comment:
Does this work when we have multiple instances of the SDLPlayData?

Careful global static const exist once per program but now these exist once per SDLPlayData instance (might be fine--just check).

This comment is for vfilters_list, afilters, nb_filters

Reply:
vfilters_list, afilters, nb_filters are not used swapped vfilters_list and nb_filters by vfilter and couldn't make the audio filter (afilters work) to enable audio at 1x using asetpts filter